### ISSUE TITLE
atomizr -i "*.sublime-snippet" -o snippets.cson - wrong bihavior

### ISSUE THREAD

**Issue by Lazerproof** [idleberg/atomizr.rb#3]
atomizr -i "*.sublime-snippet" -o snippets.cson --merge takes only first file from folder

**Comment by idleberg** [comment]
I'd love to get a look at the snippets you tried, because I can't reproduce this

**Comment by Lazerproof** [comment]
Hi.
I'm sorry for so late response. 
I'm trying to convert snippets from this repo
https://github.com/evanmcd/SublimeProcessWireSnippetsAdvanced

**Comment by idleberg** [comment]
I tried to convert these and got a warning about duplicate triggers. The triggers are used to name the files, hence they should be unique.

In part this seems to be an oversight by the author of those snippets, e.g. `config->roleTemplateID.sublime-snippet` uses `config-userTemplateID` as trigger (and content), but that's already used in `config-userTemplateID.sublime-snippet` (see my [pull request](https://github.com/evanmcd/SublimeProcessWireSnippetsAdvanced/pull/1) for details!)

Another conflict are snippets that are available twice, with and without parameters. Take `input-whitelist().sublime-snippet` and `input-whitelist(key,value).sublime-snippet`, for example. Personally, I'd only use the one with arguments and edit the snippet to easily delete the arguments (e.g. `input->whitelist(${1:\$${2:key},\$${3:value}});`) when used. Otherwise, you'll have to edit the triggers to avoid duplicates.

There's a command-line flag `--skip-duplicates` to ignore duplicate triggers, but for better user-experience, I think it's worth editing the Sublime Text snippets as described above

**Comment by Lazerproof** [comment]
Thanks for your effort.
But I have just discovered that it takes only first file from folder not only with these snippets from repo that was mentioned above. I tried the same scenario with snippets from your examples:
`https://github.com/idleberg/sublime-applescript`

---

$ atomizr -i "snippets2/*.sublime-snippet" -o snippets.cson --merge

Atomizr, version 0.19.1
The MIT License
Copyright (c) 2015, 2016 Jan T. Sott

Reading snippet file "snippets2/Calendar.Add event.sublime-snippet"
Using default scope ".source.applescript"
Writing "snippets.cson"

Atomized 1 file, created 1
Finished in 0.00901 seconds

---

I'm on windows, so maybe this issue somehow relative to OS?
